src/data/frontend-user-data.ts

```diff
@@ -0,0 +1,35 @@
+import { HomeAssistant } from "../types";
+
+/**
+ * all frontend component storage data associated with current user
+ */
+export const fetchAllFrontendUserData = async (
```

Comment:
Let's not include this one, I might even remove support from it from the backend. It will contain too much data eventually.

Reply:
Fine. But a WS api can accept list of key as input will be useful someday.